libs/configuration/src/lib/configuration/configuration.component.ts

```diff
@@ -1,11 +1,75 @@
 import { CommonModule } from '@angular/common';
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, ReactiveFormsModule } from '@angular/forms';
+import { Project } from '@angular-cli-gui/shared/data';
+import { JsonObject } from '@angular-devkit/core/src/json/utils';
+import { FormlyFieldConfig, FormlyModule } from '@ngx-formly/core';
+import { FormlyJsonschema } from '@ngx-formly/core/json-schema';
+import { FormlyMaterialModule } from '@ngx-formly/material';
+import { combineLatest, map, Subject, switchMap, tap } from 'rxjs';
+
+import { WorkspaceSettingsService } from '../services';
 
 @Component({
   selector: 'cli-configuration',
   standalone: true,
-  imports: [CommonModule],
+  imports: [
+    CommonModule,
+    FormlyModule,
+    FormlyMaterialModule,
+    ReactiveFormsModule,
+  ],
   templateUrl: './configuration.component.html',
   styleUrls: ['./configuration.component.css'],
 })
-export class ConfigurationComponent {}
+export class ConfigurationComponent implements OnInit {
+  private formFields$ = new Subject<FormlyFieldConfig[]>();
+  private formData$ = new Subject<JsonObject>();
+  private projectName = '';
+
+  formly$ = combineLatest({
+    data: this.formData$,
+    field: this.formFields$,
+  });
```

Comment:
Pretty sure combineLatest with object is deprecated

Reply:
they just realest it last year. did not know they deprecated it.
anyway, add the map to do the same